RetailCoder.VBE/Inspections/QuickFixes/QuickFixHelper.cs

```diff
@@ -43,41 +47,36 @@ public int InsertModuleLinesAfter(int lineNumber, string[] newContent)
         /// Replaces a string/line in a Code Module and returns the line number
         /// of the replaced line.
         /// </summary>
-        public int ReplaceModuleLine(int line, string newContent)
+        public int ReplaceModuleLine(int lineNumber, string newContent)
         {
-            _module.ReplaceLine(line, newContent);
-            return line;
+            _module.ReplaceLine(lineNumber, newContent);
+            return lineNumber;
         }
-        public string ReplaceStringAtIndex(int lineNumber, string toReplace, string replacement, int startIndex)
+        /// <summary>
+        /// Replaces a the TerminalNode Text value found at the location specified by the 
+        /// TerminalNode context.
+        /// </summary>
+        public void ReplaceTerminalNodeTextInCodeModule(ITerminalNode terminalNode, string replacement)
```

Comment:
Thanks - I would not have known if "Replace Token" would be sufficiently clear to other contributors.  Shorter names are always better if no meaning is lost.